tools/firmware-utils/src/mkzyxelzldfw.c

```diff
@@ -0,0 +1,835 @@
+/*
+ * Copyright (C) 2020 Vincent Wiemann <vw@derowe.com>
+ *
+ * This program is derived from ZyXEL's GPL U-Boot code.
+ * Copyright (C) 2011-2011 ZyXEL Communications, Corp.
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 as published
+ * by the Free Software Foundation.
```

Comment:
Please add an SPDX header file done here: f705008d7e6967dbdba040e97b2b77360d9c7eff

Reply:
Done.

Reply:
The license boilerplates were also removed in many recent commits, e.g. f2e86d54c42e2af2c8ebd1ca4d881f6452edbd9b